k8s-yaml/chp.yaml

```diff
@@ -42,10 +42,11 @@ spec:
           - name: TZ
             value: Europe/Helsinki
       tolerations:
-      - key: app
-        value: jupyterhub
-        operator: Equal
-        effect: NoSchedule
+      - {key: app, value: jupyterhub,             operator: Equal, effect: NoSchedule}
+      - {key: frontend,                           operator: Exists, effect: NoSchedule}
```

Comment:
This probably isn't necessary; at least in my current thinking there's no "frontend" role